src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Common/Microsoft.Azure.Search.Common.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Common types needed by the Azure Search .NET libraries. This is not the package you are looking for; It is only meant to be used as a dependency.</Description>
     <AssemblyTitle>Microsoft Azure Search Common Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Search.Common</AssemblyName>
-    <VersionPrefix>5.0.1</VersionPrefix>
```

Comment:
We recommend using `AssemblyVersion` instead of `VersionPrefix`
@shahabhijeet FYI